src/connect-widget.jsx

```diff
@@ -35,7 +35,7 @@ const ConnectWidget = () => {
 
   const launchConnectBtn = (
       // When the button is clicked, we call Connect's open method in order to display the modal
-      <button type="button" disabled={loading || !config} onClick={() => open(config)}>
+      <button type="button" className=".button" disabled={loading || !config} onClick={() => open(config)}>
```

Comment:
You don't need the period here
```suggestion
      <button type="button" className="connectButton" disabled={loading || !config} onClick={() => open(config)}>
```